src/lib/blockchain/insuranceOracle.ts

```
@@ -39,13 +39,13 @@ export async function evaluateAndTriggerPayouts(): Promise<PayoutResult[]> {
-            // 2. Ping Oracle for weather data at the specific GPS coordinates
-            const precipitation = await fetchNOAAWeatherData(policy.latitude, policy.longitude);
+            // 2. Ping Oracle for weather probability at the specific GPS coordinates
+            const twapProbability = await fetchNOAAWeatherProbability(policy.latitude, policy.longitude);
 
-            // 3. Check parametric condition: precipitation > 1.0 inches
-            if (precipitation > 1.0) {
+            // 3. Check parametric condition: TWAP rain probability > 60%
+            if (twapProbability > 60) {
                 // 4. Trigger autonomous payout (mocked blockchain interaction)
-                // In production: await contract.reportWeatherAndTrigger(policy.eventId, Math.round(precipitation * 100));
+                // In production: await contract.reportWeatherAndTrigger(policy.eventId);
```

ESLint

[error] Replace `················` with `········`

(prettier/prettier)